.idea/libraries/Dart_Packages.xml

```diff
@@ -0,0 +1,316 @@
+<component name="libraryTable">
```

Comment:
Can we remove this file from the PR? It looks like it's not covered by the .gitignore, but it likelt should be.

Reply:
I don't think it should be here but I'm not sure where it came from or how it (or the others) should be managed.